package.json

```diff
@@ -51,6 +51,7 @@
     "@types/node": "10.12.18",
     "codelyzer": "5.0.1",
     "core-js": "3.1.3",
+    "cross-var": "1.1.0",
```

Comment:
I am not sure if we need a new library dependency to support this workflow. Further, I don't see it being actively maintained/developed (no check-ins in last 2 years)

Reply:
Ok, I'm working on removing this dependency.

Reply:
I made following changes:
* Move `ng-packagr` **prod** and **dev** configs into different files and remove `cross-var` which is no more needed now.  
* Remove cleaning before prod build because `ng-packagr` takes care of that, by default `deleteDestPath=true`: https://github.com/ng-packagr/ng-packagr/blob/master/src/ng-package.schema.json#L14  
* Modify development guide to reflect these latest changes.

But seems that my branch has problems, this includes also ESLint commit. Trying to fix that.

Reply:
My branch seems ok though. So changes are done and ready for review.